### PR TITLE
fix typo in balanced_resource_allocation

### PR DESCRIPTION
**Pull request by yarntime**
fix typo in balanced_resource_allocation.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37422)

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

File: plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func fractionOfCapacity(requested, capacity int64) float64 {
 
 // BalancedResourceAllocation favors nodes with balanced resource usage rate.
 // BalancedResourceAllocation should **NOT** be used alone, and **MUST** be used together with LeastRequestedPriority.
-// It calculates the difference between the cpu and memory fracion of capacity, and prioritizes the host based on how
+// It calculates the difference between the cpu and memory fraction of capacity, and prioritizes the host based on how
```
Comment (resouer) [inline plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: Can you rebase & re-push your commit to re-trigger CI? Then I'd like to review.

**Comment by davidopp** [comment]
LGTM, but I think you need to do what @resouer said

**Comment by yarntime** [comment]
@resouer , @davidopp , I'll combine this pr with other closely related prs later, thanks.

**Pull request closed without merging**